autoload/go/statusline.vim

```diff
@@ -53,12 +53,22 @@ function! go#statusline#Show() abort
 
   " only update highlight if status has changed.
   if status_text != s:last_status
-    if status.state =~ "success" || status.state =~ "finished" || status.state =~ "pass"
-      hi goStatusLineColor cterm=bold ctermbg=76 ctermfg=22
-    elseif status.state =~ "started" || status.state =~ "analysing" || status.state =~ "compiling"
-      hi goStatusLineColor cterm=bold ctermbg=208 ctermfg=88
-    elseif status.state =~ "failed"
-      hi goStatusLineColor cterm=bold ctermbg=196 ctermfg=52
+    if (has("nvim"))
+      if status.state =~ "success" || status.state =~ "finished" || status.state =~ "pass"
+        hi goStatusLineColor cterm=bold guibg=#5fd700 guifg=#005f00
+      elseif status.state =~ "started" || status.state =~ "analysing" || status.state =~ "compiling"
+        hi goStatusLineColor cterm=bold guibg=#ff8700 guifg=#870000
+      elseif status.state =~ "failed"
+        hi goStatusLineColor cterm=bold guibg=#ff0000 guifg=#5f0000
+      endif
+    else
```

Comment:
You don't need an `if`/`else` here, I think? You can just add `guifg` and `guibg` to the `hi [..]` statements. That will also make it work for gVim and Vim with `termguicolors`.

Reply:
Thanks for your input. I will make that change